packages/docbook_xsl.rb

```diff
@@ -1,63 +1,51 @@
   end
 
   def self.install
 
-    xsl_version = '1.79.2'
+    xsl_version = '1.79.1'
     xsl_stylesheets = "xsl-stylesheets-#{xsl_version}"
     docbook_xsl = "docbook-xsl-#{xsl_version}"
 
     system "install -v -m755 -d #{CREW_DEST_PREFIX}/share/xml/#{xsl_stylesheets} &&
             cp -v -R . #{CREW_DEST_PREFIX}/share/xml/#{xsl_stylesheets}/"
     system "install -v -m644 -D README #{CREW_DEST_PREFIX}/share/doc/#{docbook_xsl}/README.txt &&
             install -v -m644 RELEASE-NOTES* NEWS* #{CREW_DEST_PREFIX}/share/doc/#{docbook_xsl}"
-    system "xmlcatalog --noout --add 'rewriteSystem' \
-                       'http://docbook.sourceforge.net/release/xsl/#{xsl_version}' \
-                       '#{CREW_PREFIX}/share/xml/docbook/#{xsl_stylesheets}' \
-                #{CREW_PREFIX}/etc/xml/catalog &&
-            xmlcatalog --noout --add 'rewriteURI' \
-                       'http://docbook.sourceforge.net/release/xsl/#{xsl_version}' \
-                       '#{CREW_PREFIX}/share/xml/docbook/#{xsl_stylesheets}' \
-                #{CREW_PREFIX}/etc/xml/catalog &&
-            xmlcatalog --noout --add 'rewriteSystem' \
-                       'http://docbook.sourceforge.net/release/xsl/current' \
-                       '#{CREW_PREFIX}/share/xml/docbook/#{xsl_stylesheets}' \
-                #{CREW_PREFIX}/etc/xml/catalog &&
-            xmlcatalog --noout --add 'rewriteURI' \
-                       'http://docbook.sourceforge.net/release/xsl/current' \
-                       '#{CREW_PREFIX}/share/xml/docbook/#{xsl_stylesheets}' \
-                #{CREW_PREFIX}/etc/xml/catalog"
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/xml"
-    system "mv #{CREW_PREFIX}/etc/xml/catalog #{CREW_DEST_PREFIX}/etc/xml/"
+    system 'cat << \'EOF\' > ./catalog.xml
+<?xml version="1.0" encoding="utf-8"?>
+<catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
+  <!-- XML Catalog file for DocBook XSL Stylesheets vsnapshot_9899 -->
+  <rewriteURI uriStartString="http://docbook.sourceforge.net/release/xsl/current/" rewritePrefix="./"/>
+  <rewriteSystem systemIdStartString="http://docbook.sourceforge.net/release/xsl/current/" rewritePrefix="./"/>
+  <rewriteURI uriStartString="http://docbook.sourceforge.net/release/xsl/snapshot_9899/" rewritePrefix="./"/>
+  <rewriteSystem systemIdStartString="http://docbook.sourceforge.net/release/xsl/snapshot_9899/" rewritePrefix="./"/>
+  <delegatePublic publicIdStartString="-//OASIS//ENTITIES DocBook XML" catalog="file:///usr/local/etc/xml/docbook"/>
+  <delegatePublic publicIdStartString="-//OASIS//DTD DocBook XML" catalog="file:///usr/local/etc/xml/docbook"/>
+  <delegateSystem systemIdStartString="http://www.oasis-open.org/docbook/" catalog="file:///usr/local/etc/xml/docbook"/>
+  <delegateURI uriStartString="http://www.oasis-open.org/docbook/" catalog="file:///usr/local/etc/xml/docbook"/>
+</catalog>
+EOF'
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/xml/"
```

Comment:
You can get rid of this line if you include the `-D` switch option in the install command below.